pallets/parachain-system/proc-macro/src/lib.rs

```diff
@@ -31,7 +31,7 @@ mod keywords {
 struct Input {
 	runtime: Path,
 	block_executor: Path,
-	check_inherents: Option<Path>,
+	check_inherents: Path,
```

Comment:
Leaving this as an `Option` seems fine, as it'd be backwards compatible. It's just that if the `Option` is `None` we'd want to drop in an empty `CheckInherents` implementation (to avoid all runtimes having to implement one, especially if it's deprecated)